plugins/catalog-backend/src/search/DefaultCatalogCollator.ts

```diff
@@ -45,7 +46,8 @@ export class DefaultCatalogCollator {
   protected filter?: GetEntitiesRequest['filter'];
   protected readonly catalogClient: CatalogApi;
   public readonly type: string = 'software-catalog';
-  public readonly visibilityPermission = catalogEntityReadPermission;
+  public readonly visibilityPermission: Permission =
```

Comment:
This property (and the ones in the factories here and in TechDocs) will be narrowed to type `ResourcePermission`, but in a separate PR. This annotation just stops the new narrower type from propagating to the api-report yet.